neptune-client-api.md

```diff
@@ -0,0 +1,56 @@
+
+
+```python
+from neptune.session import Session
+
+session = Session("token=")
+```
+
+
+```python
+project = session.get_project("neptune-ml/Ships")
+```
+
+
+```python
+exp = project.create_experiment( # create experiment, with params and tags
+    params={
+    "lr__C": 1000,
+    "epochs": 10,
+    "shuffle": true
+}, 
+    tags=["solution-6", "stacking"],
+    send_hardware_metrics=False, # default = True
+    run_monitoring_thread=False, # default = True
+    upload_sources_files=["main.py"], # default = this file
```

Comment:
@lukasz-walkiewicz I am wondering, whether some kind of magic auto discovery could be possible here.
In case of a more elaborate project, listing everything would not really be feasible.
Sacred has something like this https://sacred.readthedocs.io/en/latest/collected_information.html#source-code

Reply:
Looks interesting, I'll look into it.